docs/changelogs/CHANGELOG-0.10.md

```diff
@@ -12,12 +12,12 @@ Version 0.10 won't be supported anymore. Instead, we introduced version 1.0 whic
 - [#1798](https://github.com/epiphany-platform/epiphany/issues/1798) - Additional alerts for Prometheus
 - [#1355](https://github.com/epiphany-platform/epiphany/issues/1355) - Updating cloud based OS images - configuration required for Azure RHEL LVM images
 - [#2081](https://github.com/epiphany-platform/epiphany/issues/2081) - Replace Skopeo with Crane
-- [#1323](https://github.com/epiphany-platform/epiphany/issues/1323) - Documentation how to handle data in Opendistro for Elasticsearch
```

Comment:
Does it make sense to update old CHANGELOGS? I think we should keep them as they are

Reply:
A mistake. Corrected now.